src/native/libs/System.Globalization.Native/pal_normalization.m

```diff
@@ -75,16 +75,23 @@ int32_t GlobalizationNative_NormalizeStringNative(NormalizationForm normalizatio
             return 0;
         }
 
-        int32_t index = 0, dstIdx = 0, isError = 0;
+        // Calling with empty or null destination buffer to get the required buffer size.
+        if (lpDst == NULL || cwDstLength == 0)
+        {
+            return (int32_t)[normalizedString length];
+        }
```

Comment:
I think this part is no longer necessary, but I prefer to keep it so that it is clear what happens when destination buffer is 0.